docs/framework/wcf/feature-details/using-the-nethttpbinding.md

```diff
@@ -39,7 +39,7 @@ ms.assetid: fe134acf-ceca-49de-84a9-05a37e3841f1
         </binding>  
       </netHttpBinding>  
     </bindings>  
-    <!- ... -->   
+    <!-- ... -->   
```

Comment:
You're using ... in other places without the XML comment, so I would do the same here.